data/campaigns/Heir_To_The_Throne/scenarios/23_Test_of_the_Clans.cfg

```diff
@@ -308,6 +308,9 @@
             speaker=Sir Alric
             message= _ "Aye."
         [/message]
+#ifdef EASY
+        {NARRATOR_MESSAGE (_ "If you reduce Lord Bayar, Sir Ruga, Sir Daryn, or Sir Alric to 0HP, you'll receive a reward!")}
```

Comment:
Should use a typographic apostrophe in `you’ll`. Other than that, I like the change.

Reply:
Looks good to me as well.

Reply:
Thanks, fixed and force pushed.

While here I wonder if there are other places in the novice/rookie level campaigns where we could add such hints?